docs/source/delta-managed-commit.md

```diff
@@ -0,0 +1,88 @@
+---
+description: Learn about Delta Managed Commits.
+---
+
+# Delta Managed Commits
+
+.. warning:: This feature is available in preview in <Delta> 4.0. Since this feature is still in preview, it can undergo major changes. Furthermore, a table which has this preview feature enabled cannot be written to by future Delta releases until
```

Comment:
This is not rendering as expected in the github markdown viewer.

Reply:
It should render correctly in the docs: 
<img width="1561" alt="image" src="https://github.com/delta-io/delta/assets/125324092/7278529b-3910-4f67-9fb5-1868ede12b2c">